api/main_endpoints/util/redis-client.js

```diff
@@ -0,0 +1,40 @@
+const { createClient } = require('redis');
+const logger = require('../../util/logger');
+
+class MockRedisClient {
+  constructor() {
+    this.store = new Map();
+  }
+
+  async connect() {
+  }
+
+  async set(key, value, options = {}) {
+    this.store.set(key, value);
+    return true;
+  }
+
+  async get(key) {
+    return this.store.get(key) || null;
+  }
+
+  async del(key) {
+    return this.store.delete(key) ? true : false;
+  }
+}
+
+const redisHost = process.env.REDIS_HOST;
+
+const client = redisHost
+  ? createClient({ url: `redis://${redisHost}:6379` })
```

Comment:
can we change the environment variable to be the entire url, not just the host? that way we dont need to format the string before calling createClient

the variable could be renamed to `REDIS_HOST_URL`